src/index.html

```diff
@@ -10,13 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"class="logo">
+        <img
```

Comment:
The `alt` attribute for the logo image should describe the image content. 'navigationlogo' is not descriptive enough. Consider something like 'Company Logo'.